roles/vendor/certbot/tasks/main.yml

```diff
@@ -1,21 +1,17 @@
 ---
+- include_tasks: cloudflare-plugin.yml
+- name: Update apt-get repo and cache
+  apt: update_cache=yes cache_valid_time=3600
 - name: Install installation deps
   apt:
     name: software-properties-common
     state: present
-- name: Certbot ppa added
```

Comment:
this will not work on ubuntu 20.04+

Reply:
Then make it work. 👍🏼

Reply:
For example: https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal (click Wildcard)
This uses Snap, as per official recommendations. Also do note that the docs explicitly state that pip-based installation is not officially supported and recommended.